README.md

```diff
@@ -36,11 +36,34 @@ end
 
 Then to auth with Cronofy you would navigate to `/auth/cronofy`
 
+### Service Accounts
+
+Service Accounts allow you to use one set of credentials to access an entire organizations calendar service. This works with Google Apps, Office 365 and Exchange.
+
+They can be used alongside standard end-user auth.
+
+Typical configuration.
+
+```ruby
+Rails.application.config.middleware.use OmniAuth::Builder do
+
+  provider :cronofy_service_account, ENV["CRONOFY_CLIENT_ID"], ENV["CRONOFY_CLIENT_SECRET"], {
+    scope: "service_account/manage_accounts",
+    delegated_scope: "read_account read_events create_event delete_event",
+  }
+
+end
+```
+
+More information in Service Accounts documentation (link to follow).
+
 ## Configuration
 
 Configurable options
 
-* `scope`: A space-separated list of permissions you want to request from the user. See the [API Authorization documentation](http://www.cronofy.com/developers/api#authorization) for a full list of available permissions.
+* `scope`: A space-separated list of permissions you want to request for the authorization. See the [API Authorization documentation](http://www.cronofy.com/developers/api#authorization) for a full list of available permissions.
```

Comment:
The end user scopes are only relevant to the delegated scopes so probably better to only link on on that. Its a pain we haven't got a link for the service account scopes yet. 

I'd go with something like

> A space-separated list of permissions you want to request for the service account. See the Service Account documentation for the full list.